simpleclient/src/main/java/io/prometheus/client/Histogram.java

```diff
@@ -247,6 +254,20 @@ public Timer startTimer() {
     return noLabelsChild.startTimer();
   }
 
+  /**
+   * Executes runnable code and observes a duration of how long it took to run
```

Comment:
Sentence should end with a period.

I'd also mention in here that this is intended for use with Java 8+